poolscript/script_test.go

```diff
@@ -132,6 +132,32 @@ func FuzzWitnessSpendDetection(f *testing.F) {
 	})
 }
 
+// TestHasAnnex tests whether hasAnnex can correctly identify
+// the presence of an annex element within a witness stack.
+func TestHasAnnex(t *testing.T) {
+	t.Parallel()
+
+	// The annex element is the last element and includes the correct prefix.
```

Comment:
nit: 81 char line

Reply:
I'm don't think I understand what you mean. [Line 140](https://github.com/ffranr/pool/blob/8ff3a0e3ecbd14bdd534ff206cee845ccf6b6831/poolscript/script_test.go#L140) is 74 chars by my count.

Reply:
Ah I see what's up, you're counting tabs as 8 characters.